### PR TITLE
add add-ons log verbosity level option

### PR DESCRIPTION
**Pull request by moelsayed**
https://github.com/rancher/rancher/issues/10185

### CODE REVIEW

**Comment by alena1108** [comment]
@moelsayed can you change image to be rancher/k8s:v1.8.2-rancher1

File: infra-templates/k8s/34/rancher-compose.yml [hunk 1]
```diff
@@ -84,6 +84,16 @@ catalog:
...
+  - variable: ADDONS_LOG_LEVEL
+    label: Log level for Kubernetes Add-ons
+    description: Log level for the Kubernetes Add-ons. The numbers for severity levels INFO, WARNING, ERROR, and FATAL are 0, 1, 2, and 3, respectively
+    default: 2
+    type: enum
+    options:
+      - 0
+      - 1
+      - 2
+      - 3
```
Comment (alena1108) [inline infra-templates/k8s/34/rancher-compose.yml]: is 3 the highest number across all addon services?

Comment (moelsayed) [reply infra-templates/k8s/34/rancher-compose.yml]: @alena1108 Yes. They are all using [glog](https://github.com/golang/glog) which has only 4 levels.

Comment (moelsayed) [reply infra-templates/k8s/34/rancher-compose.yml]: @alena1108 Updated all related versions.

Comment (alena1108) [reply infra-templates/k8s/34/rancher-compose.yml]: I guess we are mixing level and verbosity here according to https://github.com/rancher/kubernetes-package/pull/108/files#diff-9ca9c9f2447011552247630cc47afe84R25

Comment (moelsayed) [reply infra-templates/k8s/34/rancher-compose.yml]: correction: https://github.com/kubernetes/kubernetes/issues/35054

**Comment by alena1108** [comment]
LGTM. @deniseschannon could you give it a second glance

**Comment by deniseschannon** [comment]
LGTM

**Pull request merged**